doc/content/devices/best-practices/_index.md

```diff
@@ -0,0 +1,62 @@
+
+## Synchronization, Backoff, and Jitter
+
+Synchronization of devices happens if end devices respond to a large-scale external event - for example, hundreds of end devices that are connected to the same power source and the power is switched off and on again, or hundreds of end devices that are connected to the same gateway, and the firmware of the gateway needs to be updated.
+
+Let’s take an example device that starts in Join mode when it powers on, and reverts to Join mode after being disconnected from the network. There are hundreds of such devices in a field, and one gateway that covers this field.
+
+The power source for the devices is switched on, and the gateway immediately receives the noise of hundreds of simultaneous Join requests. LoRaWAN gateways can deal quite well with noise, but this is just too much, and the gateway can’t make any sense of it. No Join requests are decoded, so no Join requests are forwarded to the network and no Join requests are accepted.
+
+If the retry interval is a fixed duration, e.g. 10 seconds, the gateway again receives the noise of hundreds of simultaneous Join requests, and still can’t make anything of it. This continues every 10 seconds after that, and the entire site stays offline.
+
+### Jitter
+
+This situation can be improved by using jitter. Instead of sending a Join request every 10 seconds, the devices send a Join request 10 seconds after the previous one, plus or minus a random duration of 0-20% of this 10 seconds. This jitter percentage needs to be truly random, because if all devices use the same pseudorandom number generator, they will still be synchronized, as they will all pick the same “random” number.
+
+With these improved devices, the Join requests will no longer all be sent at exactly the same time, and the gateway will have a better chance of decoding the Join requests.
+
+### Backoff
+
+But what if you have another site with thousands of these devices? Then the 10 seconds between Join messages may not be enough. This is where backoff comes in. Instead of having a delay of 10s±20%, you increase the delay after each attempt, so you do the second attempt after 20s±20%, the third after 30s±20%, and you keep increasing the delay until you have, say, 1h±20% between Join requests.
+
+An implementation like this prevents persistent failures of sites and the network as a whole and helps speed up recovery after outages.
+
+## Additional Best Practices
+
+- Save device parameters between regular power cycles
+- Use a true random number generator (especially for jitter and channel hopping)
+- Use [OTAA]({{< ref "/reference/glossary#otaa" >}}) instead of [ABP]({{< ref "/reference/glossary#abp" >}})
+- Optimize [data rate]({{< ref "/reference/glossary#data-rate" >}})
+- Try to use [ADR]({{< ref "/reference/glossary#adr" >}}) for non moving devices, and ADR for moving devices
```

Comment:
```suggestion
- Try to use [ADR]({{< ref "/reference/glossary#adr" >}}) for immobile devices
```
What does `, and ADR for moving devices` mean?

Also, see the related wording/explanation in the spec (1.0.3 in this case)
![2020-09-23-23:34:53-screenshot](https://user-images.githubusercontent.com/12877905/94072215-906b7e00-fdf5-11ea-999a-2fdead4babc4.png)